ui/src/pages/namespace/Explorer/Tree/index.tsx

```diff
@@ -90,75 +100,85 @@ const ExplorerPage: FC = () => {
       <div className="p-5">
         <Card>
           {showTable && (
-            <Dialog open={dialogOpen} onOpenChange={setDialogOpen}>
-              <Table>
-                <TableBody>
-                  <div className="flex justify-between gap-5 p-2">
-                    <Input
-                      data-testid="queryField"
-                      className="sm:w-60"
-                      value={query}
-                      onChange={(e) => {
-                        setQuery(e.target.value);
+            <>
+              <div className="flex justify-between gap-5 p-2">
+                <Input
+                  data-testid="queryField"
+                  className="sm:w-60"
+                  value={query}
+                  onChange={(e) => {
+                    setQuery(e.target.value);
+                  }}
+                  placeholder={t("pages.explorer.tree.list.filter")}
+                />
+              </div>
+              <DropdownMenuSeparator />
```

Comment:
Ah, thanks for the hint with the instance list page. 👍 
Because of the error before, I thought I should not insert the component in the table at all, but I understood now that in the TableCell it is allowed.

So while I was at it, I also put the Input field in a TableCell, because it is visually the same result. (But needed to overwrite the hover.) Hope you like that change...

Although I like the style how the NoResult is used in the Instances page, I think it wood look confusing when using too many parameters (in this case: icon, text, button are different between NoResult and NoSearchResult). 
So I would rather keep a separate component for NoSearchResult and NoResult. 
Do you agree?